SetupDataPkg/Docs/Overview/Overview.md

```diff
@@ -162,32 +163,43 @@ firmware volume.
 - **BDS**: Project MU BDS will provide specific event signals and platform entrypoints that are customized for DFCI settings.
 - **DFCI**: DFCI framework will be used to accept and validate incoming configuration against platform identity associated
 certificates.
+- **MFCI**: The MFCI framework will be used to check what mode the system is in, manufacturing mode or customer mode.
 - **Settings Manager**: Settings manager together with DFCI framework would apply the configuration data through platform
 configured settings providers.
 - **Policy Manager**: Policy manager controls the policy publication and revoke. Silicon policy and platform configuration
 should all conform to policy setter and getter APIs.
+- **Profile Manager**: ConfProfileMgrDxe validates and enforces profiles (collections of configuration settings for
+different use cases) when the system is in MFCI Customer Mode.
 
 ### Platform and Silicon Code
 
 - **Silicon Drivers**: Silicon code needs to be updated to pull policy settings from silicon policy data when needed.
 - **Platform Policy Drivers**: Platform owners will first create PEI modules to populate default silicon policy into Policy
 managers provided by Project MU.
-- **Platform YAML/XML Configurations**: Platform owners should then design the configuration YAML/XML files. This would expose
+- **Platform YAML Configurations**: Platform owners should then design the configuration YAML files. This would expose
 certain configuration "knobs" from silicon policy to be configurable through setup variable flow.
 - **Platform Settings Providers**: Accordingly, platform owners will develop modules to parse the configuration data and
-translate the exposed configurations in YAML/XML file to/from standard silicon policies through the interface of Settings
+translate the exposed configurations in YAML file to/from standard silicon policies through the interface of Settings
 Provider (see example from [Project MU](../../../Common/MU/DfciPkg/Library/DfciSampleProvider/DfciSampleProvider.c)).
 
+## Profiles
+
+See the [Profiles doc](../Profiles/Overview.md) for details.
+
 ## Configuration Related UEFI Boot Flow
 
 ![UEFI Build](Images/uefi_boot_flow.png)
 
+### Settings Update Boot Flow
+
 - Formatted update configuration data from USB, serial port or OS application will first be stored to UEFI variable storage,
 followed by a system reboot.
-- Upon next boot, stored formatted variable will be authenticated and decoded by DFCI framework. Decoded configuration
-data will be dispatched to corresponding platform authored setting provider.
+- Upon next boot, stored formatted variable will be authenticated (if the
```

Comment:
Could "Upon next boot" be changed to "Upon the reboot"?

"stored formatted variable" would be clearer as "formatted configuration data from UEFI Variable storage"

Reply:
Sure, will change.